code_samples/simple_language_plugin/src/com/simpleplugin/SimpleAnnotator.java

```diff
@@ -1,37 +1,41 @@
 package com.simpleplugin;
```

Comment:
Thank you for contributing!
1) Can you describe the defect your changes fix? (Or the feature it adds?)
2) Would you please push a set of changes to SimpleAnnotator.java based on the current revision of the file? There were substantial changes to the annotate method checked-in on 11-Feb-2020. The baseline you have used here predates those changes.

Reply:
Hello,
1.
- In the first commit I improved the code by : 
  * removing some indentation level
  * doing some extract and call to remove duplicate code
- In the second commit I fixed annotator test
2. I'll rebase my fork